Makefile

```diff
@@ -0,0 +1,19 @@
+GO ?= go
+
+PKGS ?= ./...
+
+BIN := bin/main
+
+.PHONY: cli get vet test
+
+cli:
+	mkdir -pv $(dir $(BIN)) && $(GO) build -o $(BIN) ./cli
+
+get:
+	$(GO) get $(ARGS) $(PKGS)
+
+vet:
+	$(GO) vet $(ARGS) $(PKGS)
+
+test:
+	$(GO) test $(ARGS) $(PKGS)
```

Comment:
Could you initiate separate issues for the makefile and interface changes? They are not related and should be considered as separate pull requests.

Reply:
Sure thing! I made issues #10, #11. Please let me know if you'd like to me to prepare a PR as well.